src/ui/src/wds/WdsCheckbox.vue

```diff
@@ -55,6 +60,15 @@ function onChange(event: InputEvent) {
 	opacity: 40%;
 	cursor: not-allowed;
 }
+
+.WdsCheckbox--invalid .WdsCheckbox__checkbox {
+	border-color: var(--wdsColorOrange5);
+}
+
+.WdsCheckbox--invalid .WdsCheckbox__label {
+	color: var(--wdsColorOrange5);
+}
```

Comment:
Could you try to avoid using new classes and use CSS only?

```suggestion
.WdsCheckbox:has(input[aria-invalid="true"]) .WdsCheckbox__checkbox {
	border-color: var(--wdsColorOrange5);
}

.WdsCheckbox:has(input[aria-invalid="true"]) .WdsCheckbox__label {
	color: var(--wdsColorOrange5);
}
```